apps/designer/app/designer/designer.tsx

```diff
@@ -296,7 +309,13 @@ export const Designer = ({ config, project, page }: DesignerProps) => {
         </Workspace>
       </Main>
       <SidePanel gridArea="sidebar" isPreviewMode={isPreviewMode}>
-        <SidebarLeft publish={publish} />
+        <SidebarLeft
+          publish={publish}
+          project={project}
+          pages={pages}
```

Comment:
We can't pass all of this to sidebar component, because this pattern won't let us scale the panels. Each feature panel should get the data on its own, e.g. using nano state